test/integration/annotation_spec.mjs

```diff
@@ -140,7 +141,7 @@ describe("Checkbox annotation", () => {
           );
           for (const selector of selectors) {
             await page.click(selector);
-            page.waitForTimeout(10);
+            await waitForTimeout(10);
```

Comment:
I don't know if or how the old code here worked, but the new code should at least be correct now until we hopefully eventually remove it in the scope of #17656.